locale/en/LC_MESSAGES/django.po

```diff
@@ -1866,6 +1911,24 @@ msgstr "Category requires a name."
 msgid "COURSE_CONFIG_ERROR_MODULE_REQUIRES_KEY"
 msgstr "Module requires a key."
 
+#: edit_course/operations/configure.py
+#, fuzzy, python-brace-format
+#| msgid "PUBLISH_RESPONSE_NON_200 -- {status_code}"
+msgid "PUBLISH_RESPONSE_NON_200 -- {status_code}"
+msgstr "Publishing responded with status {response.status_code}"
+
+#: edit_course/operations/configure.py
+#, python-brace-format
+msgid "PUBLISH_ERROR_JSON_PARSER_FAILED -- {e}, {text}"
+msgstr ""
+"Failed to load publish response JSON: {e}:\n"
```

Comment:
"publish" refers to the endpoint that is used to publish (the changes to) a course. I couldn't quickly come up with a better message, so I put that. Maybe "publishing" instead of "publish"?

Reply:
"Failed to load the publishing response JSON:"